bin/docker-entrypoint

```diff
@@ -4,9 +4,10 @@ set -e
 worker() {
   WORKERS_COUNT=${WORKERS_COUNT:-2}
   QUEUES=${QUEUES:-queries,scheduled_queries,celery,schemas}
+  WORKER_EXTRA_OPTIONS=${WORKER_EXTRA_OPTIONS:}
```

Comment:
@jasonthomas Is this a legitimate shell script?

Reply:
No, it is throwing a bad substitution error. It should be `WORKER_EXTRA_OPTIONS=${WORKER_EXTRA_OPTIONS:-}` instead.